### DISCUSSION TITLE
Installing newest HO version fails

### DISCUSSION THREAD

**Discussion by emar9** [ho-dev/HattrickOrganizer#911] [Q&A]
I tried to install the newest HO (HO-4.1.3310.2.dmg) but the system says that I've to install the newest Java 14 Version. I updated the newest Java in my Java Control Panel. I have now the Java Version 8 Update 281 on my Mac installed. Where can I find the Java 14 version? I couldn't find it on the java.com page. 

My operating system: macOS 10.13.6 (17G14042)

**Comment by akasolace** [comment]
https://adoptopenjdk.net/
